src/app/app-interactive-map/components/loader/loader.component.html

```diff
@@ -12,9 +12,18 @@
 <!--See the License for the specific language governing permissions and-->
 <!--limitations under the License.-->
 <mat-dialog-content class="mat-primary" fxLayout="colum" fxFlexAlign="center center">
-  <div class="map-progress">
-    <mat-spinner class="mat-primary"></mat-spinner>
-    <h3 class="map-progress-text">Calculating flux distribution...</h3>
+  <div class="map-progress" >
+    <mat-spinner class="mat-primary" *ngIf="(loading | async)"></mat-spinner>
+    <h3 class="map-progress-text" *ngIf="(loading | async)">Calculating flux distribution...</h3>
+    <div class="error-box" *ngIf="!(loading | async)">
+      <h3 class="map-progress-text-error">An error ocurred. Please try again later.</h3>
```

Comment:
> An error ocurred. Please try again later.

It's short and to the point, but maybe a little bit cold? How about something a little bit friendlier:

> Oops, a technical error occurred while handling your request. We've been notified of the problem and we will investigate as soon as possible. Meanwhile, you can try to refresh the page, or give it another shot in a little while. We apologize for the inconvenience.

A bit longer so would probably look better with a smaller font and maybe a title.